redbeams-api/src/main/java/com/sequenceiq/redbeams/doc/ModelDescriptions.java

```diff
@@ -9,23 +9,23 @@
     public static final String CREATION_DATE = "Creation date / time of the resource, in epoch milliseconds";
     public static final String ENVIRONMENT_ID = "ID of the environment of the resource";
 
-    // public static class Database {
-    //     public static final String CONNECTION_URL = "JDBC connection URL in the form of jdbc:<db-type>://<address>:<port>/<db>";
-    //     public static final String CONNECTION_DRIVER = "Name of the JDBC connection driver (for example: 'org.postgresql.Driver')";
-    //     public static final String DB_ENGINE = "Name of the external database engine (MYSQL, POSTGRES...)";
-    //     public static final String DB_ENGINE_DISPLAYNAME = "Display name of the external database engine (Mysql, PostgreSQL...)";
-    //     public static final String VERSION = "Version of the Database";
-    //     public static final String USERNAME = "Username to use for the jdbc connection";
-    //     public static final String PASSWORD = "Password to use for the jdbc connection";
-    //     public static final String ORACLE = "Oracle specific properties";
-    //     public static final String NAME = "Name of the RDS configuration resource";
-    //     public static final String STACK_VERSION = "(HDP, HDF)Stack version for the RDS configuration";
-    //     public static final String RDSTYPE = "Type of RDS, aka the service name that will use the RDS like HIVE, DRUID, SUPERSET, RANGER, etc.";
-    //     public static final String CONNECTOR_JAR_URL = "URL that points to the jar of the connection driver(connector)";
-    //     public static final String DATABASE_REQUEST_CLUSTER_NAME = "requested cluster name";
-    //     public static final String DATABASE_CONNECTION_TEST_RESULT = "result of RDS connection test";
-    //     public static final String DATABASE_REQUEST = "unsaved RDS config to be tested by connectivity";
-    // }
+    public static class Database {
+        public static final String CONNECTION_URL = "JDBC connection URL in the form of jdbc:<db-type>://<address>:<port>/<db>";
+        public static final String CONNECTION_DRIVER = "Name of the JDBC connection driver (for example: 'org.postgresql.Driver')";
+        public static final String DB_ENGINE = "Name of the external database engine (MYSQL, POSTGRES...)";
+        public static final String DB_ENGINE_DISPLAYNAME = "Display name of the external database engine (Mysql, PostgreSQL...)";
+        public static final String VERSION = "Version of the Database";
+        public static final String USERNAME = "Username to use for the jdbc connection";
+        public static final String PASSWORD = "Password to use for the jdbc connection";
+        public static final String ORACLE = "Oracle specific properties";
+        public static final String NAME = "Name of the RDS configuration resource";
+        public static final String STACK_VERSION = "(HDP, HDF)Stack version for the RDS configuration";
```

Comment:
Some of these model descriptions are unused, like STACK_VERSION, so they can be left commented out for now, if not deleted completely.